doc/Docker_deployment.md

```diff
@@ -0,0 +1,29 @@
+# Deploying a module in a local Docker development environment
+
+## Build your module
+
+See [Module_builder.md](Module_builder.md).
+
+## Obtain prerequisites
+
+1. Docker Toolbox: https://www.docker.com/toolbox
```

Comment:
Docker claims that Docker Toolbox includes VirtualBox, but since I already had it I didn't have a good way of testing.